### ISSUE TITLE
Issue with HMDB dataset training

### ISSUE THREAD

**Issue by rookie1234** [yjxiong/temporal-segment-networks#17]
Hi

I was trying to train the temporal segment networks with HMDB dataset. I see the following error. I used the extraction script mentioned in the repo to get hmdb frames. Could you please take a look at this error.

I0914 00:56:45.781002 23644 net.cpp:732]     [Update] Layer fc-action, param 1 data: 0; diff: 0.000178426
I0914 00:56:45.985848 23644 solver.cpp:616] Gradient clipping: scaling down gradients (L2 norm 43.1335 > 20) by scale factor 0.463677
F0914 00:56:46.026162 25274 data_transformer.cpp:314] Check failed: width <= datum_width (224 vs. 176) 
*** Check failure stack trace: ***

Thanks!

**Comment by yjxiong** [comment]
Please see also #6

**Comment by yjxiong** [comment]
@rookie1234 Is this solved?
